packages/in_app_purchase/in_app_purchase_platform_interface/README.md

```diff
@@ -0,0 +1,26 @@
+# in_app_purchase_platform_interface
+
+A common platform interface for the [`in_app_purchase`][1] plugin.
+
+This interface allows platform-specific implementations of the `in_app_purchase`
+plugin, as well as the plugin itself, to ensure they are supporting the
+same interface.
+
+# Usage
+
+To implement a new platform-specific implementation of `in_app_purchase`, extend
+[`InAppPurchasePlatform`][2] with an implementation that performs the
+platform-specific behavior, and when you register your plugin, set the default
+`InAppPurchasePlatform` by calling
+`InAppPurchasePlatform.setInstance(MyPlatformInAppPurchase())`.
+
```

Comment:
Let's also add doc for how to add additional features in a platform

Reply:
done